docs/README.md

```diff
@@ -49,6 +49,7 @@ ES3 support has been removed from the latest version (v3.x), if required [see fo
 
 - [Create an Issue](https://github.com/microsoft/ApplicationInsights-JS/issues/new/choose)
 - [ES3 Support](./es3_Support.md)
+- [Page Unload Event Handling](https://microsoft.github.io/ApplicationInsights-JS/docs/PageUnloadEvents.html)
```

Comment:
Update this to reference the markdown file as github publishing will provide the correct conversion